src/cmd/api/db/dao/video.go

```diff
@@ -0,0 +1,62 @@
+package dao
+
+import (
+	"database/sql"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/Sogilis/Voogle/src/cmd/api/db/models"
+)
+
+func PutVideo(db *sql.DB, video models.VideoModelUpload) (string, error) {
+	id := uuid.NewString()
+	clientId := uuid.NewString()
+
+	query := "INSERT INTO videos (id, client_id, title) VALUES ('" + id + "','" + clientId + "', ?);"
```

Comment:
Use parameters instead building the query like this.

+ We can certainly prepare the query